bin/uglifyjs

```diff
@@ -254,7 +258,17 @@ if (specified["self"]) {
     if (!options.wrap) options.wrap = "UglifyJS";
     paths = UglifyJS.FILES;
 }
-if (paths.length) {
+if (specified["in-situ"]) {
+    if (specified["reduce-test"] || specified["self"]) fatal("incompatible options specified");
```

Comment:
`-o, --output <file>` should also be fatal when used with `--in-situ`.

Reply:
Good catch &minus; done in https://github.com/mishoo/UglifyJS/pull/4758/commits/55bc4249e038f0dc8d9089d9fd156c4abe5a6099

Just noticed that we won't be able to handle non-inlined `--source-map`
https://github.com/mishoo/UglifyJS/blob/01fa430a3e20098361ff8d35306ba1c31fea514c/bin/uglifyjs#L212
but I guess we'll cross that brodge if/when we get there.